src/Utilities.js

```diff
@@ -39,7 +44,11 @@ export function isConnected(node) {
   if (nativeValue !== undefined) {
     return nativeValue;
   }
-
+  // Optimization: It's significantly faster here to try to use `contains`,
+  // especially on Edge/IE/
+  if (document.contains(node)) {
```

Comment:
Given the `document.contains` polyfill above, would `node.ownerDocument.contains(node)` work in all cases here? I'd rather not polyfill that function if it isn't necessary.